app/poros/npc.rb

```diff
@@ -5,7 +5,7 @@ class NPC
               :ancestry,
               :sub_ancestry,
               :background,
-              :class,
+              :character_class,
```

Comment:
I changed this because it was messing some things up calling `.class` on the NPC object.